### DISCUSSION TITLE
Is there a way to change the "Show Recipes" Keybind?

### DISCUSSION THREAD

**Discussion by DaCooladeMan** [shedaniel/RoughlyEnoughItems#1486] [Q & A]
I'm having trouble with it all the time. when in creative mode, I search for a block but when my search has R in it, my mouse happens to be hovering over a block in the creative menu when I press R, so it shows the Recipe of the block, and then I have to start the search over again. Is there a way to change the key bind? I don't see it in Mod Menu or Controls.

**Comment by gegentan** [comment]
I have the same issue, but also with keybinds U and A.
